contracts/burner/src/contract.rs

```diff
@@ -59,7 +63,9 @@ pub fn query<S: Storage, A: Api, Q: Querier>(
     _deps: &Extern<S, A, Q>,
     _msg: QueryMsg,
 ) -> StdResult<Binary> {
-    Err(generic_err("You can only use this contract for migrations"))
+    Err(StdError::generic_err(
```

Comment:
Can you please add a section for 0.8 -> 0.9 migrations?

I think this is one obvious change. The other is not needed `?` after the various API calls (as they no longer return a `Result`). With that doc change, good to merge.